### PR TITLE
Bug 1774936: fix VM Wizard cloud init and other style/usability issues

### PR DESCRIPTION
**Pull request by suomiy**
- allign add ssh key
- disable remove when there are no ssh keys present
- fix color of remove ssh key
- add hidden label to ssh key input
- replace Gi units with GiB
- fix warning for custom scripts
- fix focus trapping for memory/cpu fields
- default to os id if name not available
- ask user for confirmation when leaving the vm wizard

### CODE REVIEW

**Comment by suomiy** [comment]
@matthewcarleton

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import * as _ from 'lodash';
 import { connect } from 'react-redux';
+import { match as RouterMatch } from 'react-router';
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx]: nit: I think this line moved by mistake from L10 ?

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx]: ah yes, fixed

File: frontend/packages/kubevirt-plugin/src/components/form/size-unit-utils.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { assureEndsWith } from '@console/shared/src';
+
+export enum BinaryUnit {
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/form/size-unit-utils.tsx]: We already have a units file, can we use the public component's units here ?
https://github.com/openshift/console/blob/master/frontend/public/components/utils/units.js#L22

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/form/size-unit-utils.tsx]: These ones are typed and you do not need an index to reference them

**Comment by yaacov** [comment]
Except for the units question /lgtm

**Pull request merged**